tests/test_plotImageSubtractionCutouts.py

```diff
@@ -300,13 +300,19 @@ def test_pickle(self):
         """Test that the task is pickleable (necessary for multiprocessing).
         """
         config = plotImageSubtractionCutouts.PlotImageSubtractionCutoutsTask.ConfigClass()
-        config.size = 63
+        config.size = [63]
         cutouts = plotImageSubtractionCutouts.PlotImageSubtractionCutoutsTask(config=config,
                                                                               output_path="something")
         other = pickle.loads(pickle.dumps(cutouts))
         self.assertEqual(cutouts.config.size, other.config.size)
         self.assertEqual(cutouts._output_path, other._output_path)
 
+    def test_config_size_type(self):
+        """Ensure that the 'size' paramter in config is a list and not a number
+        """
+        config = plotImageSubtractionCutouts.PlotImageSubtractionCutoutsTask.ConfigClass()
+        self.assertIsInstance(config.size, lsst.pex.config.listField.List)
```

Comment:
This is already checked during config validation when setting values. For example, if you change the line above back to `config.size = 63`, that test will fail at that line because the field is now a `ListField`.

The test you should add instead is one that makes a plot with two sizes of cutouts. We can tweak their layout a bit (for example, there's no need to repeat the labels on each row), and then be sure that that layout is kept as other code changes. You can also use the output of that test to add to the examples in the restructured text `doc/` directory.